var/spack/repos/builtin/packages/trinity/package.py

```diff
@@ -48,6 +48,7 @@ class Trinity(MakefilePackage):
     depends_on("bowtie2")
     depends_on("jellyfish")
     depends_on("salmon")
+    depends_on("perl+threads")
 
     def build(self, spec, prefix):
         make
```

Comment:
Does this package actually work? `make` is a function, I didn't think Python would let you call a function without parentheses...